src/Service/Terminal/Service_Terminal.re

```diff
@@ -49,6 +59,29 @@ module Sub = {
             arguments: [],
           };
 
+        let onEffect = eff =>
+          switch (eff) {
+          | ReveryTerminal.ScreenResized(screen)
+          | ReveryTerminal.ScreenUpdated(screen) =>
+            dispatch(TerminalScreenUpdated({id: params.id, screen}))
+          | ReveryTerminal.CursorMoved(cursor) =>
+            dispatch(TerminalCursorMoved({id: params.id, cursor}))
+          // TODO: Handle output
```

Comment:
This will be coming in the next PR, for handling focus / input keys in the terminal.